api/openvpn-rw/connection-history

```diff
@@ -37,6 +38,13 @@ if time_interval != "today" and time_interval != "last_week" and time_interval !
     sys.stderr.write('Time interval must be one of "today", "last_week" or "last_month"\n')
     exit(1)
 
+DATABASE_PATH = '/var/lib/nethserver/openvpn/connections.db'
+database_exists = os.path.isfile(DATABASE_PATH)
+
+if not database_exists:
+    syslog.syslog(syslog.LOG_ERR, 'Error: {} does not exist. Please execute /etc/e-smith/events/actions/nethserver-openvpn-create-connections-db'.format(DATABASE_PATH))
+    sys.exit(1)
```

Comment:
Why not reuse the same logic of line 38 and writing to stderr? stederr will be already available inside the log.
```suggestion
if not database_exists:
    sys.stderr.write('Error: {} does not exist. Please execute /etc/e-smith/events/actions/nethserver-openvpn-create-connections-db'.format(DATABASE_PATH))
    sys.exit(1)
```